Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/BasicActionBuilder.java

```diff
@@ -50,7 +49,7 @@
     private boolean promptable;
 
     private boolean telemetrable;
-    private TelemetryParameter telemetryParameter;
+    private String actionString;
```

Comment:
May need to find a better name for this parameter, it contains the service & operation name, but we can not get the hint from the name